src/app/stacktrace_win_dlg.h

```diff
@@ -60,7 +60,11 @@ class StraceDlg : public QDialog, private Ui::errorDialog
             "</p></font>"
             "<br/><hr><br/>"
             "<p align=center><font size=4>"
-            "qBittorrent version: " VERSION "<br/>"
+            "qBittorrent version: " VERSION
```

Comment:
>For example?

nevermind, doesn't seem to work.

>I think (64-bit) should appear only in Windows version.

No problem for me.
However adding a condition & string in `version.pri` could be better, rather than modify these 2 files.